.env.dist

```diff
@@ -2,3 +2,16 @@ QUARTZY_TOKEN=
 ELABFTW_HOST_URL=https://elab.local:3148/api/v2/
 ELABFTW_API_KEY=
 CATEGORIES=["Antibody", "Plasmid", "-80 boxes"]
+# Optional: Proxy settings
+# uncomment and adjust the following variables.
+
+#HTTP_PROXY=http://127.0.0.1:8080
```

Comment:
use http://proxy.example.org:3128

3128 is the default port of many proxy software